lib/matplotlib/artist.py

```diff
@@ -1331,7 +1313,17 @@ def get_setters(self):
         Get the attribute strings with setters for object.  e.g., for a line,
         return ``['markerfacecolor', 'linewidth', ....]``.
```

Comment:
Might as well fix this:
```suggestion
        Get the attribute strings with setters for object.

        For example, for a line, return ``['markerfacecolor', 'linewidth', ....]``.
```

Reply:
yup